### ISSUE TITLE
Remove SNMP path 

### ISSUE THREAD

**Issue by wimpers** [openvstorage/framework#678]
Since the SNMP path isn't working and we are pushing stats to Redis.

**Comment by wimpers** [comment]
@khenderick any idea how much work this would be

**Comment by khenderick** [comment]
@wimpers, shouldn't take more than a few hours.

**Comment by khenderick** [comment]
Fixed by #1002, packaged in openvstorage-2.7.4-rev.4088.62c4f94

**Comment by JeffreyDevloo** [comment]
### Steps
- Installed ovs packages
- Installed ovs cluster

Checked the following files:
- /opt/OpenvStorage/config/templates/upstart/ovs-snmp
- /opt/OpenvStorage/juju/charms/trusty/openvstorage/*
- /opt/OpenvStorage/ovs/extensions/snmp/*
- /etc/init/ovs-snmp.conf

Checked if python-snmp is installed:
Package:
- python-pysnmp4
- python-pysnmp4-apps
- python-pysnmp4-mibs

### Output

init file removed:

```
root@ovs-node1:~# cat /etc/init/ovs-snmp.conf
cat: /etc/init/ovs-snmp.conf: No such file or directory
```

ovs-snmp python files removed from ovs:

```
root@ovs-node1:~# ls /opt/OpenvStorage/ovs/extensions/snmp/
ls: cannot access /opt/OpenvStorage/ovs/extensions/snmp/: No such file or directory
```

Template file removed:

```
root@ovs-node1:~# cat /opt/OpenvStorage/config/templates/upstart/ovs-snmp
cat: /opt/OpenvStorage/config/templates/upstart/ovs-snmp: No such file or directory
```

Juju removed:

```
root@ovs-node1:~# ls /opt/Openvstorage/juju
ls: cannot access /opt/Openvstorage/juju: No such file or directory
```

Check if dependencies were installed:
erlang-snmp is installed by the openvstorage-hc package

```
root@ovs-node1:~# dpkg -l | grep snmp
ii  erlang-snmp                        1:16.b.3-dfsg-1ubuntu2.1                                 amd64        Erlang/OTP SNMP applications
```

### Test result

There is still a SNMP package installed by OVS. The package (erlang-smp) is installed due to rabbitmq-server depending on erlang-nox which depends on erlang-snmp

```
rabbitmq-server
  Depends: adduser
  Depends: erlang-nox (>= 1:13.b.3)
  Depends: esl-erlang
  Depends: logrotate
erlang-nox
  ...
  Depends: erlang-snmp
  ...
```

Test passed.

### Packages
- openvstorage                       2.7.4-rev.4161.5f26491-1                                 amd64        openvStorage
- openvstorage-backend               1.7.4-rev.770.ae31d34-1                                  amd64        openvStorage Backend plugin
- openvstorage-backend-core          1.7.4-rev.770.ae31d34-1                                  amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps       1.7.4-rev.770.ae31d34-1                                  amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin         1.2.2-rev.38.dcc3b76-1                                   amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                  2.7.4-rev.4161.5f26491-1                                 amd64        openvStorage core
- openvstorage-hc                    1.7.4-rev.770.ae31d34-1                                  amd64        openvStorage Backend plugin HyperConverged
- openvstorage-sdm                   1.6.4-rev.390.89e2ad4-1                                  amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps               2.7.4-rev.4161.5f26491-1                                 amd64        openvStorage Web Applications
